src/utils/init/frameworks.js

```diff
@@ -0,0 +1,23 @@
+const { listFrameworks } = require('@netlify/framework-info')
+
+const getFrameworkInfo = async ({ siteRoot, nodeVersion }) => {
+  const frameworks = await listFrameworks({ projectDir: siteRoot, nodeVersion })
+  if (frameworks.length !== 0) {
+    const [
```

Comment:
Are we only considering the first element of the array on purpose (i.e. the `framework-info` lib is only returning one framework for the time being)?

Reply:
It might detect a few frameworks - the first one has higher priority based on the order in https://github.com/netlify/framework-info/blob/32c58bb8de2f86874a61bdc7abab059c4c638f6c/src/frameworks/main.js#L4.

Reply:
I wouldn't mind having a comment here just highlighting that (we pick the first as its the highest priority). Not critical though, happy to move forward if you feel like it doesn't make sense @erezrokah 👍

Reply:
That makes sense. I'll add it

Reply:
Done in https://github.com/netlify/cli/pull/1836/commits/e1a410e10e1fc4390dc7c58ae8b786ec5caf9a54